site/_includes/macros/navigation-tree-sections.njk

```diff
@@ -16,7 +16,7 @@ and marks it as active.
   {% if item.sections %}
     <button class="navigation-tree__link button" data-expandable aria-expanded="{{ item.active }}">
       <div class="navigation-tree__icon">
-        {{ icon('arrow-right', {label: 'toggle'}) }}
+        {{ icon('arrow-right', {label: 'toggle', hidden: true}) }}
```

Comment:
Should this label be localized?

Reply:
I've hidden this one and used the button outside it instead.